ui/app/pages/swaps/build-quote/build-quote.js

```diff
@@ -113,7 +113,7 @@ export default function BuildQuote({
   const toTokenIsNotEth =
     selectedToToken?.address &&
     selectedToToken?.address !== ETH_SWAPS_TOKEN_OBJECT.address;
-
+  const occurances = Number(selectedToToken?.occurances || 0);
```

Comment:
Made this small change.  `NaN < 2` is `false`, so we need to have `0` as fallback.

Reply:
It seems a bit inappropriate to show this message before the user has selected anything though :thinking:

Reply:
Nothing is shown until the user does have something selected though:

<img width="497" alt="EmptySwap" src="https://user-images.githubusercontent.com/46655/109232012-23398b80-778d-11eb-9b4a-63befffb0fa7.png">

Maybe I'm missing something?

Reply:
Oh I see - because `toTokenIsNotEth` evaluates to `false` in that case.

Nevermind, this all makes sense. I thought it worked differently from reading the code at first.